docs/topics/user_permissions.md

```diff
@@ -0,0 +1,51 @@
+# Users and permissions
+
+
+## What user permissions are defined in IoTP?
+
+
+In IoTP there are defined Services and SubServices. SubServices exists into Services.
+Tipicaly a Service represents a smartcity and all of the SubServices represents the verticals of that smartcity.
+
+Users are created into Services. The same user name could be used across different Services to represent different Users.
+i.e. adm1 could exists in "SmartCity" Service and adm1 could be another user for "SmartGondor" Service
+
+Roles are created into Services. By default all Services created into IoT platform are created with the following Roles:
+- ServiceCustomer: Role for a normal user of the Service, with standard capabilities.
+- SubServiceCustomer: Role for a normal user in a SubService, with standard capabilities.
+- SubServiceAdmin: Role for an administrator user in a SubService, with full capabilities.
+
+There is one and unique role common for all Services:
```

Comment:
If we mention other roles this role should be mentioned, since applies to current service admin